auth/src/main/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivity.java

```diff
@@ -83,7 +85,7 @@ protected void onCreate(Bundle savedInstanceState) {
         if (logoId == AuthUI.NO_LOGO) {
             findViewById(R.id.logo_layout).setVisibility(View.GONE);
         } else {
-            ImageView logo = (ImageView) findViewById(R.id.logo);
+            ImageView logo = (ImageView) findViewById(R.id.logo_layout);
```

Comment:
Yes, it they wrapped ImageView into LinearLayout which has this id now and this new id is used in tests. I can't wrap ImageView like that so I changed its id to avoid conflict in tests.